src/components/App/functions.js

```diff
@@ -37,6 +43,19 @@ export function createSocket (app) {
   return socket;
 }
 
+export function queryMessages (app) {
```

Comment:
this function name is vague enough to me that we should be commenting what it's doing.